addon/-private/helpers.js

```diff
@@ -1,9 +1,13 @@
 import Ember from 'ember';
 import Ceibo from 'ceibo';
+import require from 'require';
+import $ from '-jquery';
 
 const { assert, get, isPresent } = Ember;
 
-import $ from '-jquery';
+const emberTestHelpers = require.has('ember-test-helpers') ?
```

Comment:
I'm pretty sure this won't work for `ember-test-helpers` `0.6.x` and earlier. `ember-test-helpers` has never exported `wait` from the [module index](https://github.com/emberjs/ember-test-helpers/blob/v0.6.3/lib/ember-test-helpers.js), it's always only been available as the default export of `ember-test-helpers/wait`. I think this is the matrix:

### <= 0.6.x
* `require('ember-test-helpers').wait` doesn't exist
* `require('ember-test-helpers/wait')` exists
* `require('ember-test-helpers/settled')` does not exist

### 0.7.x
* `require('ember-test-helpers').wait` does not exists
* `require('ember-test-helpers/wait')` exists
* `require('ember-test-helpers/settled')` exists, but I think kinda by accident since `settled` is part of the new `@ember/test-helpers` API

I think you should probably just do

```javascript
const wait = require.has('ember-test-helpers/wait') ? require('ember-test-helpers/wait') : null;
```

and then check for `wait` being truthy.

Reply:
Ah, good point...thx!